src/GUI/EFCorePowerTools/EFCorePowerTools.csproj

```diff
@@ -293,6 +296,7 @@
       <HintPath>..\lib\Microsoft.Data.Tools.Utilities.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Microsoft.EntityFrameworkCore.Design, Version=2.2.1.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL" />
```

Comment:
Is this really needed? If so, use 2.2.0

Reply:
Hm, that might have slipped in there thanks to ReSharper. I'll remove it.